tarpc/src/transport/mod.rs

```diff
@@ -0,0 +1,84 @@
+use std::io::{self, Read, Write};
+use std::time::Duration;
+
+/// A factory for creating a listener on a given address.
```

Comment:
Removed the associated type `Addr` from `Dialer` but still haven't resolved the address issue in comments. Is it just a documentation thing, then?